.gitlab-ci.yml

```diff
@@ -214,7 +214,21 @@ notifications:
         - var/spack/repos/builtin/packages/singularity/*
         - var/spack/repos/builtin/packages/singularityce/*
   script:
-    - pip install -r bluebrain/notifications/requirements.txt
-    - git log -n 3 >> bluebrain/notifications/apptainer_singularity.txt
-    - echo -e '{noformat}\n\nThank you!' >> bluebrain/notifications/apptainer_singularity.txt
-    - python bluebrain/notifications/jira_ticket.py --project HELP --summary "Please run the spack permissions fix scripts" --description-file bluebrain/notifications/apptainer_singularity.txt
+    - apt-get update
+    - apt-get install -y jq wget
+    - echo "Finding ID of the atomic build"
+    - ATOMIC_BUILD_ID=$(wget --header "Content-Type:application/json" --header "PRIVATE-TOKEN:${API_READ_TOKEN}" -O- "${CI_API_V4_URL}/projects/${CI_PROJECT_ID}/pipelines/${CI_PIPELINE_ID}/bridges" | jq '.[] | select(.name == "atomic_build") | .downstream_pipeline.id')
+    - echo "Atomic build pipeline ID: ${ATOMIC_BUILD_ID}"
+    - set +e
+      # TODO: API_READ_TOKEN should be replaced by one not tied to my user account
+    - wget --header "Content-Type:application/json" --header "PRIVATE-TOKEN:${API_READ_TOKEN}" -O- "${CI_API_V4_URL}/projects/${CI_PROJECT_ID}/pipelines/${ATOMIC_BUILD_ID}/test_report"
+    - |
+      wget --header "Content-Type:application/json" --header "PRIVATE-TOKEN:${API_READ_TOKEN}" -O- "${CI_API_V4_URL}/projects/${CI_PROJECT_ID}/pipelines/${ATOMIC_BUILD_ID}/test_report" | jq -e '.test_suites[] | select(.name == "externals") .test_cases[] | select(.classname == "singularity" or .classname == "singularityce" or .classname == "apptainer")'
```

Comment:
Final test: modify an irrelevant package and see if this gets triggered? I don't see anything that qualifies if the package should be a successful / skipped / failed test.

Reply:
I was under the impression that the test just had to be present, but you're right, skipped tests should be ignored. Added that now.

https://bbpgitlab.epfl.ch/hpc/personal/heeren/cicd/-/pipelines/141632 demonstrates the jq syntax, experiments were done here: https://bbpgitlab.epfl.ch/hpc/personal/heeren/cicd/-/blob/fetch-tests-through-api/.gitlab-ci.yml (feel free to re-run the pipeline on branch `fetch-tests-through-api`, it doesn't take long).